pkg/overlay/server_test.go

```diff
@@ -19,211 +19,202 @@ import (
 )
 
 func TestServer(t *testing.T) {
-	ctx := testcontext.New(t)
-	defer ctx.Cleanup()
-
-	planet, err := testplanet.New(t, 1, 4, 1)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer ctx.Check(planet.Shutdown)
-
-	planet.Start(ctx)
-	// we wait a second for all the nodes to complete bootstrapping off the satellite
-	time.Sleep(2 * time.Second)
-
-	satellite := planet.Satellites[0]
-	server := satellite.Overlay.Endpoint
-	// TODO: handle cleanup
-
-	{ // Lookup
-		result, err := server.Lookup(ctx, &pb.LookupRequest{
-			NodeId: planet.StorageNodes[0].ID(),
-		})
-		require.NoError(t, err)
-		require.NotNil(t, result)
-		assert.Equal(t, result.Node.Address.Address, planet.StorageNodes[0].Addr())
-	}
-
-	{ // BulkLookup
-		result, err := server.BulkLookup(ctx, &pb.LookupRequests{
-			LookupRequest: []*pb.LookupRequest{
-				{NodeId: planet.StorageNodes[0].ID()},
-				{NodeId: planet.StorageNodes[1].ID()},
-				{NodeId: planet.StorageNodes[2].ID()},
-			},
-		})
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1, StorageNodeCount: 8, UplinkCount: 1,
```

Comment:
This one changed from StorageNodeCount 4 to 8. Since all the others kept the same parameters, it's worth asking if this was intended?

Reply:
Unintentional

Reply:
fixed